app/src/main/java/com/orange/ouds/app/ui/tokens/Token.kt

```diff
@@ -38,7 +38,16 @@ data class Token<T>(val name: String, val relativeName: String, val value: @Comp
                 if (value.isUnspecified) {
                     stringResource(id = R.string.app_tokens_color_unspecified_label)
                 } else {
-                    stringResource(id = R.string.app_tokens_colorFormat_label, value.value.toString(16).substring(2, 8).uppercase())
+                    stringResource(
+                        id = R.string.app_tokens_colorFormat_label,
+                        value.value.toString(16).let { hex ->
+                            when {
+                                hex == "0" -> "00000000" // Transparent color
+                                hex.startsWith("ff") -> hex.substring(2, 8) // No need to show alpha
+                                else -> hex.substring(0, 8)
+                            }
+                        }.uppercase()
```

Comment:
It seems there is something wrong with our computation of the the hex value (and it was already the case before). For instance opacity transparent is displayed as `#FFFF00` although it should be `#00FFFFFF` according to `opacityTransparentDark` in `OrangeColorOpacitySemanticTokens`:

<img width="432" height="960" alt="Screenshot_20251202_113356" src="https://github.com/user-attachments/assets/b1839266-a14d-42c6-8866-a4ed7e006059" />

If I am correct, simply using `value.toArgb().toHexString().uppercase()` in all cases fixes the issue (and we can filter the two first characters is they are equal to `FF` if you want).